ThunderCloud/QuizTextSelectionViewController.swift

```diff
@@ -24,7 +24,7 @@ class QuizTextSelectionViewController: TableViewController {
 		guard let question = question else { return }
 		
 		tableView.allowsMultipleSelection = question.limit > 1
-		tableView.contentInset = UIEdgeInsetsMake(0, 0, 44+32, 0)
+		tableView.contentInset = UIEdgeInsets.init(top: 0, left: 0, bottom: 44+32, right: 0)
```

Comment:
Do you need `.init`?

Reply:
Nope, definitely not! Swift migration tool -.-